Cargo.toml

```
@@ -28,22 +28,24 @@ license = "Apache-2.0"
 description = "Extend the capabilities of DataFusion to support additional data sources via implementations of the `TableProvider` trait."
 
 [workspace.dependencies]
-arrow = "55.0.0"
-arrow-array = { version = "55.2.0" }
-arrow-flight = { version = "55.0.0", features = [
+arrow = "56.0.0"
+arrow-array = { version = "56.0.0" }
+arrow-flight = { version = "56.0.0", features = [
   "flight-sql-experimental",
-  "tls",
+  "tls-ring",
 ] }
-arrow-ipc = { version = "55.0.0" }
-arrow-schema = { version = "55.2.0", features = ["serde"] }
-arrow-json = "55.0.0"
-arrow-odbc = { version = "16.0.2" }
-datafusion = { version = "49", default-features = false }
-datafusion-expr = { version = "49" }
-datafusion-federation = { version = "0.4.7" }
-datafusion-ffi = { version = "49" }
-datafusion-proto = { version = "49" }
-datafusion-physical-expr = { version = "49" }
-datafusion-physical-plan = { version = "49" }
+arrow-ipc = { version = "56.0.0" }
+arrow-schema = { version = "56.0.0", features = ["serde"] }
+arrow-json = "56.0.0"
+arrow-odbc = { version = "20.0.0" }
+datafusion = { version = "50", default-features = false }
+datafusion-expr = { version = "50" }
+#datafusion-federation = { version = "0.4.7" }
+# TODO: update
+datafusion-federation = { git = "https://github.com/datafusion-contrib/datafusion-federation/", rev = "5202ac748f8f5de054a02d79a23ad1284133f5f7" }
+datafusion-ffi = { version = "50" }
+datafusion-proto = { version = "50" }
+datafusion-physical-expr = { version = "50" }
+datafusion-physical-plan = { version = "50" }
 datafusion-table-providers = { path = "core" }
 duckdb = { version = "=1.3.0", package = "spiceai_duckdb_fork" } # Forked to add support for duckdb_scan_arrow, pending: https://github.com/duckdb/duckdb-rs/pull/488
```

```suggestion
duckdb = { version = "=1.3.1", package = "spiceai_duckdb_fork" } # Forked to add support for duckdb_scan_arrow, pending: https://github.com/duckdb/duckdb-rs/pull/488
```

I've published `1.3.1` with Arrow 56